scripts/generate-llms.ts

```diff
@@ -0,0 +1,145 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+
+import libraries from '~/react-native-libraries.json';
+import { type DataAssetType, type LibraryDataEntryType } from '~/types';
+import { parseGitHubUrl } from '~/util/parseGitHubUrl';
+
+type LibraryRecord = {
+  library: LibraryDataEntryType;
+  repoUrl: string;
+  repoName: string;
+  description: string;
+};
+
+const OUTPUT_PATH = path.resolve('public', 'llms.txt');
+const DATASET_PATH = path.resolve('assets', 'data.json');
+const INTRODUCTION = [
+  '# reactnative.directory',
+  'Browse the full catalog at https://reactnative.directory for fresh React Native libraries.',
+];
+
+const PLATFORM_LABELS = {
+  android: 'Android',
+  expoGo: 'Expo Go',
+  fireos: 'Fire OS',
+  horizon: 'Meta Horizon',
+  ios: 'iOS',
+  macos: 'macOS',
+  tvos: 'tvOS',
+  visionos: 'visionOS',
+  vegaos: 'Vega OS',
+  web: 'Web',
+  windows: 'Windows',
+} as const;
```

Comment:
Let's retain the spit between platforms and compatibility terms, and reuse existing helpers from:
* https://github.com/react-native-community/directory/blob/main/components/Filters/helpers.ts

We can also move them to global `/utils` if needed.